modules/_labs/combobox/react/stories/stories.tsx

```diff
@@ -24,20 +56,12 @@ class Autocomplete extends React.Component<
   };
 
   render() {
-    const autocompleteResult = (textModifier: string) => (
-      <MenuItem onClick={action(`Clicked Result ${textModifier}`)}>
-        Result{' '}
-        <span>
-          num<span>ber</span>
-        </span>{' '}
-        {textModifier}
-      </MenuItem>
-    );
+    const Results = this.props.useComplexMenu ? complexAutocompleteResult : autocompleteResult;
     return (
       <Combobox
         {...this.props}
         autocompleteItems={Array.apply(null, Array(this.state.currentText.length))
-          .map((x: any, i: string) => autocompleteResult(i))
+          .map((x: any, i: string) => Results(i))
```

Comment:
Is `Results` intended to be a function or a component? The capitalization hints at component while the type signature suggests function.

Reply:
@NicholasBoll closed this PR in favor of a draft one https://github.com/Workday/canvas-kit/pull/530